experiments/conductor/cmd/runner/utilities.go

```diff
@@ -267,7 +268,7 @@ type CommandConfig struct {
 }
 
 // Helper function to execute a command with timing and logging
-func executeCommand(cfg CommandConfig, out *strings.Builder, stderr *strings.Builder) error {
+func executeCommand(cfg CommandConfig) (string, string, error) {
```

Comment:
If we want to make the timeout configurable, it would probably be more concise to just pass Options in to the executeCommand function and let it (here) determine the over-ride. Right now we are passing timeout through the CommandConfig struct, which would require us to extract the field at every call site and set it in the CommandConfig before calling us.

Reply:
done. Added a second commit to set global timeout and pass it thorough.